### PR TITLE
Match against react translations

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Compare current translation strings against React/RN & update angular to match. 
- Since we didn't publish v2.3.0, I've updated the changelog to include the translation string updates as part of v2.3.0; originally this was planned as it's own release.

### CODE REVIEW

**Comment by jeffvg** [comment]
@huayunh when you get time can you take a peek at this

File: login-workflow/src/translations/chinese.ts [hunk 1]
```diff
@@ -77,33 +78,34 @@ export const pxbAuthChineseTranslations: PxbAuthTranslations = {
...
             VALIDATING: '证实',
-            INVALID_TITLE: '错误',
+            INVALID_TITLE: '错误！',
             INVALID_DESCRIPTION: '注册帐户时出错。',
         },
         ENTER_EMAIL: {
-            TITLE: '创建一个帐户',
-            INSTRUCTIONS: '要注册Eaton帐户，请在下面输入所需信息。您需要验证您的电子邮件地址才能继续。.',
+            TITLE: '创建账号',
+            INSTRUCTIONS: '无法完成注册。请点击“继续”退出。',
```
[the remark this answers is not in the document]
Comment (huayunh) [reply login-workflow/src/translations/chinese.ts:86]: Where is this used?

[the remark this answers is not in the document]
Comment (huayunh) [reply login-workflow/src/translations/chinese.ts:86]: As far as I can tell, this is the only place where the before and after commit indicate two totally different things

**Review by daileytj — APPROVED** [review]

File: login-workflow/src/translations/chinese.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (emclaug2) [reply login-workflow/src/translations/chinese.ts:86]: > Where is this used?

This is used within the Create Account (not via invite) workflow; whenever the user has to enter their email.

**Pull request merged**